sycl/doc/design/DeviceIf.md

```diff
@@ -249,7 +329,7 @@ constexpr static auto if_architecture_is(T fnTrue, Args ...args) {
   }
 }
 
-} // namespace ext::intel::experimental
+} // namespace ext::oneapi::exprimental
 } // namespace sycl
 ```
 
```

Comment:
This comment is for the sentence below that says:

> The only supported targets are `spir64_x86_64` and the new `intel_gpu_*` GPU device names.

I think that sentence should be updated to include the "nvidia" and "amd" device names.